examples/plugins/job-pre-post/sleep/avocado_job_sleep.py

```diff
@@ -0,0 +1,21 @@
+import time
+import logging
+
+from avocado.core.settings import settings
+from avocado.plugins.base import JobPrePost
+
+
+class Sleep(JobPrePost):
+
+    name = 'sleep'
+    description = 'Sleeps for a number of seconds'
+
+    def run(self, job):
+        log = logging.getLogger("avocado.app")
+        seconds = settings.get_value(section="plugins.job.sleep",
+                                     key="seconds",
+                                     key_type=int,
+                                     default=3)
+        for i in range(seconds):
```

Comment:
I know it's just small difference, but I'm allergic to this. Please use `xrange` instead of `range` when you don't need to create the list of items :wink:.

Also how about `xrange(1, seconds + 1)` to avoid the need to add the 1 in each step?